maplibre-adapter/src/androidTest/java/com/amplifyframework/geo/maplibre/MapViewTestActivityTest.kt

```diff
@@ -15,41 +15,33 @@
 
 package com.amplifyframework.geo.maplibre
 
-import androidx.test.core.app.ActivityScenario.launch
-
-import com.amplifyframework.auth.AuthCategory
-import com.amplifyframework.auth.cognito.AWSCognitoAuthPlugin
-import com.amplifyframework.testutils.sync.TestCategory
+import androidx.test.core.app.launchActivity
 
 import com.mapbox.mapboxsdk.maps.MapboxMap
+
+import kotlinx.coroutines.runBlocking
+
 import org.junit.Assert.assertNotNull
-import org.junit.Before
 import org.junit.Test
-import java.lang.RuntimeException
-import kotlin.coroutines.*
-import kotlinx.coroutines.runBlocking
+
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
 
 class MapViewTestActivityTest {
-    /**
-     * Configure MapLibre Adapter's auth provider.
-     */
-    @Before
-    fun setUpAuth() {
-        AmplifyMapLibreAdapter.auth = TestCategory.forPlugin(AWSCognitoAuthPlugin()) as AuthCategory
-    }
```

Comment:
i think this is good to leave here so that whenever we add new test suites in this class, we make sure to build a fresh category per test case

Reply:
I moved it to the `MapViewTestActivity`, but I'll move it back to re-create per test case